mappings/net/minecraft/world/entity/LookupContinuityGetter.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_7927 net/minecraft/world/entity/LookupContinuityGetter
```

Comment:
Imo, `net/minecraft/util/LazyIterationConsumer` as modders could possibly use this class for other purposes. This is a consumer that accepts lazily evaluated elements in an iteration.